tests/test_datapackage.py

```diff
@@ -345,6 +345,44 @@ def test_raises_if_remote_resource_url_doesnt_exist(self):
         with pytest.raises(IOError):
             datapackage.DataPackage(descriptor).resources[0].data
 
+    @httpretty.activate
+    def test_if_relative_resource_of_remote_package_has_absolute_path(self):
```

Comment:
This test appears to be testing the same as the `test_if_resource_of_remote_package_has_local_data_path`, so it can be removed.